h2o-docs/src/product/data-science/glm.rst

```diff
@@ -66,21 +67,24 @@ Defining a GLM Model
 
 -  `family <algo-params/family.html>`__: Specify the model type.
 
-   -  If the family is **gaussian**, the data must be numeric (**Real** or **Int**).
-   -  If the family is **binomial**, the data must be categorical 2 levels/classes or binary (**Enum** or **Int**).
-   -  If the family is **multinomial**, the data can be categorical with more than two levels/classes (**Enum**).
-   -  If the family is **ordinal**, the data must be categorical with at least 3 levels.
-   -  If the family is **quasibinomial**, the data must be numeric.
-   -  If the family is **poisson**, the data must be numeric and non-negative (**Int**).
-   -  If the family is **gamma**, the data must be numeric and continuous and positive (**Real** or **Int**).
-   -  If the family is **tweedie**, the data must be numeric and continuous (**Real**) and non-negative.
+   -  If the family is **gaussian**, the response must be numeric (**Real** or **Int**). (default)
+   -  If the family is **binomial**, the response must be categorical 2 levels/classes or binary (**Enum** or **Int**).
+   -  If the family is **multinomial**, the response can be categorical with more than two levels/classes (**Enum**).
+   -  If the family is **ordinal**, the response must be categorical with at least 3 levels.
+   -  If the family is **quasibinomial**, the response must be numeric.
+   -  If the family is **poisson**, the response must be numeric and non-negative (**Int**).
+   -  If the family is **negativebinomial**, the response must be numeric (**Int**).
```

Comment:
Please change this:  the response must be numeric (**Int**).
to: the response must be numeric and non-negative (**Int**).

Reply:
Fixed!